modules/carousel/jetpack-carousel.js

```diff
@@ -987,6 +987,17 @@ jQuery(document).ready(function($) {
 			if ( medium_width >= args.max_width || medium_height >= args.max_height ) {
 				return args.medium_file;
 			}
+			
+			if ( isPhotonUrl ) {
+				// args.orig_file doesn't point to a Photon url, so in this case we use args.large_file
+				// to return the photon url of the original image.
+				var largeFileIndex = args.large_file.lastIndexOf( '?' );
+				var origPhotonUrl = args.large_file;
+				if ( largeFileIndex != -1 ) {
```

Comment:
can you change this to strict operator `!==`?  our js tests are picky. 

Also please use <a href="https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#yoda-conditions">Yoda style conditional</a> for this.  Thanks!